### PR TITLE
Adds AsyncImport example and makes it the default example.

### PR DESCRIPTION
**Pull request by tedwards**
Also fixes issues #8 and #26

### CODE REVIEW

File: ImportResult.php [hunk 1]
```diff
@@ -47,7 +47,7 @@ public function __construct($importResultElement)
 		{
 		        $this->_title = (string) $importResultElement->title;
 		        $this->_message = (string) $importResultElement->message;
-		        $this->_wasSuccessful = (bool) $importResultElement["successful"];
+		        $this->_wasSuccessful = filter_var( $importResultElement["successful"], FILTER_VALIDATE_BOOLEAN);
```
Comment (brianrogers) [inline ImportResult.php]: neat... I haven't used that before.

File: samples/config.php [hunk 2]
```diff
@@ -32,6 +32,7 @@
 require_once('../ScormEngineUtilities.php');
 unset($CFG);
 
+$CFG = new StdClass();
```
Comment (brianrogers) [inline samples/config.php]: checkstyle related?

File: samples/AsyncImport.php [hunk 3]
```diff
@@ -41,30 +55,19 @@
 $ScormService = new ScormEngineService($ServiceUrl,$AppId,$SecretKey,$Origin);
 $courseService = $ScormService->getCourseService();
 
-$token = $courseService->ImportCourseAsync('mycourseid', '/path/to/course/here');
+$courseId = uniqid();
+$interstitial = $CFG->wwwroot."/AsyncStatus.php";
```
Comment (brianrogers) [inline samples/AsyncImport.php]: Is AsyncStatus.php already part of the fileset?

Comment (tedwards) [inline samples/AsyncImport.php]: Yup. Totally forgot to add those.

File: samples/config.php (hunk 2 above)
Comment (tedwards) [inline samples/config.php]: I think back in 5.4 assigning an attribute to a reference that was null started throwing a warning.  #8

**Pull request merged**